src/redux/categories/categoriesSlice.js

```diff
@@ -8,11 +8,15 @@ export const categoriesSlice = createSlice({
   name: 'categories',
   initialState,
   reducers: {
-    checkStatus: (state, action) => {
-      state.categories = action.payload === 'Under construction'
-        ? 'Under construction'
-        : state.categories;
-    },
+    // checkStatus: (state) => {
+    //   const newState = { ...state };
+    //   newState.status = 'Under Construction';
+    //   return newState;
+    // },
```

Comment:
- [ ] Great job implementing some of the changes requested by the previous reviewer. Unfortunately in updating the check status button to dispatch the right action, you commented the old code. The previous reviewer did indicate that you should remove unsed (commented) code to make your codebase clean. Kindly remove the commented lines of code. 
You can read this [article](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) on JavaScript best practices to avoid such errors in the future.